Jenkinsfile

```diff
@@ -12,7 +12,7 @@
 
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
-//@Library(value="pipeline-lib@your_branch") _
+@Library(value="pipeline-lib@skip_optane") _
```

Comment:
Need to remove this of course before landing.

Reply:
Done